File: 2271-rearrange-array-elements-by-sign/2271-rearrange-array-elements-by-sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        vector<int> neg;
        vector<int> pos;
        int n=nums.size();
        for(int i=0;i<n;i++){
            if(nums[i]>=1){
                pos.push_back(nums[i]);
            }
            else{
                neg.push_back(nums[i]);
            }
        }
        vector<int> v;
        for(int i = 0;i<n/2;i++){
            v.push_back(pos[i]);
            v.push_back(neg[i]);
        }

        return v;

        
    }
};

File: 2271-rearrange-array-elements-by-sign/2271-rearrange-array-elements-by-sign.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        pos=[]
        neg=[]
        n=len(nums)
        for i in range(n):
            if(nums[i]>0):
                pos.append(nums[i])
            else:
                neg.append(nums[i])
        a=[]
        
        for i in range(n//2):
            a.append(pos[i])
            a.append(neg[i])
        return a

        
        